src/test/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAOTest.java

```diff
@@ -623,6 +624,107 @@ void testGetDarCollectionSummaryForAdmin_ArchivedCollection() {
     assertEquals(collectionOneId, summaries.get(0).getDarCollectionId());
   }
 
+  @ParameterizedTest
+  @ValueSource(strings= {"admin", "researcher"})
+  void testGetDarCollectionSummaryDraftAndArchivedNotIncluded(String type) {
+    User user = createUser();
+    Integer userId = user.getUserId();
+
+    Dataset dataset = createDataset(userId);
+
+    Integer collectionId = createDarCollection(userId);
+
+    // Create two DataAccessRequests in the same collection
+    DataAccessRequest olderDar = createDataAccessRequest(collectionId, userId);
+    DataAccessRequest draftDar = createDataAccessRequest(collectionId, userId);
+    DataAccessRequest archivedDar = createDataAccessRequest(collectionId, userId);
+
+    // Insert dataset relations for all DARs
+    dataAccessRequestDAO.insertDARDatasetRelation(olderDar.getReferenceId(), dataset.getDatasetId());
+    dataAccessRequestDAO.insertDARDatasetRelation(draftDar.getReferenceId(), dataset.getDatasetId());
+    dataAccessRequestDAO.insertDARDatasetRelation(archivedDar.getReferenceId(), dataset.getDatasetId());
+    // draft DAR
+    dataAccessRequestDAO.updateDataByReferenceId(draftDar.getReferenceId(), draftDar.userId, new Date(), null,
+        new Date(), draftDar.getData());
+    // archived DAR
+    dataAccessRequestDAO.archiveByReferenceIds(List.of(archivedDar.getReferenceId()));
+
+    List<DarCollectionSummary> summaries = switch (type) {
+      case "admin" -> darCollectionSummaryDAO.getDarCollectionSummariesForAdmin();
+      case "researcher" -> darCollectionSummaryDAO.getDarCollectionSummariesForResearcher(userId);
+      default -> throw new IllegalArgumentException("Invalid type: " + type);
+    };
+
+    assertNotNull(summaries);
+    assertEquals(1, summaries.size());
+    DarCollectionSummary summary = summaries.get(0);
+    assertEquals(collectionId, summary.getDarCollectionId());
+
+    // Ensure the reference IDs include only the non-draft non-archived DAR
+    assertNotNull(summary.getReferenceIds());
+    assertEquals(1, summary.getReferenceIds().size());
+    assertTrue(summary.getReferenceIds().contains(olderDar.getReferenceId()));
+    assertFalse(summary.getReferenceIds().contains(draftDar.getReferenceId()));
```

Comment:
Since there's only one element you don't need to `assertFalse()`
```suggestion
    assertTrue(summary.getReferenceIds().contains(olderDar.getReferenceId()));
```